client/components/PubPreview/pubPreview.scss

```diff
@@ -220,6 +220,13 @@
 		display: inline;
 		font-weight: 500;
 	}
+	.byline-icon {
+		margin-bottom: 3px;
```

Comment:
Super-duper minor, I prefer using `position: relative` for minor offset tweaks like this. When I read margin my brain thinks "pushing other stuff away".

Reply:
`tranform: translateY(-3px)` is also a good option here